### PR TITLE
Provide Python 3 variant of the tests

### PR DESCRIPTION
**Pull request by jstasiak**
This file is just Python 2 tests with one change to a print statement.

### CODE REVIEW

File: python3/tests.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    def test_delete_event(self):
+        filename = os.path.join(self.watch_directory, "tmp")
+        open(filename, "w").write("")
+        self._get_events()
+        del event_holder[:]
+
+        os.remove(filename)
+        self._get_events()
+
+        self._print_events()
```
Comment (alexdias) [inline python3/tests.py:98]: @jstasiak Why are we printing events here?

File: python3/tests.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import unittest
+import os
+import pyinotify
+import tempfile
+import shutil
+
+event_holder = []
+
+
+class processor(pyinotify.ProcessEvent):
```
Comment (alexdias) [inline python3/tests.py:10]: @jstasiak Why is the class name not capitalised here?

File: python3/tests.py (hunk 1 above)
Comment (jstasiak) [inline python3/tests.py:98]: For debugging reasons as far as I can tell, this code is just a copy of what's already in the repository (commit https://github.com/smarkets/pyinotify-smarkets/commit/fdb18c1031eeb0789cb968bd9cbe3d0988357a25)

Comment (alexdias) [inline python3/tests.py:98]: Ah, cool

File: python3/tests.py (hunk 2 above)
Comment (jstasiak) [inline python3/tests.py:10]: This is a copy of code introduced in https://github.com/smarkets/pyinotify-smarkets/commit/fdb18c1031eeb0789cb968bd9cbe3d0988357a25

Comment (jstasiak) [inline python3/tests.py:10]: PS. I'll probably rename it when I do some cleaning up in the near future.

Comment (alexdias) [inline python3/tests.py:10]: Ah ok, that's cool!

**Comment by alexdias** [comment]
This seems alright!

**Pull request merged**